.github/workflows/share_PR_builder.yml

```
@@ -0,0 +1,71 @@
+name: Pophory Share SDK PR Builder
+
+on:
+  pull_request:
+    branches: [ develop, master ]
+
+defaults:
+  run:
+    shell: bash
+    working-directory: .
+
+jobs:
+  build:
+    name: PR Checker
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v3
+
+      - name: Gradle cache
+        uses: actions/cache@v3
+        with:
+          path: |
+            ~/.gradle/caches
+            ~/.gradle/wrapper
+          key: ${{ runner.os }}-gradle-${{ hashFiles('**/*.gradle*') }}
+          restore-keys: |
+            ${{ runner.os }}-gradle-
+      - name: set up JDK 17
+        uses: actions/setup-java@v3
+        with:
+          distribution: 'temurin'
+          java-version: 17
+
+      - name: Change gradlew permissions
+        run: chmod +x ./gradlew
+
+      - name: Add Local Properties
+        env:
+          SENTRY_DSN: ${{ secrets.SENTRY_DSN }}
+        run: |
+          echo sentryDsn=\"$SENTRY_DSN\" >> ./local.properties
+      - name: Access Firebase Service
+        run: echo '${{ secrets.GOOGLE_SERVICES_JSON }}' > ./app/google-services.json
+
+      - name: Build debug APK
+        run: ./gradlew assembleDebug --stacktrace
```

```suggestion
        run: ./gradlew :app:assembleDebug --stacktrace
```

멀티 모듈이어서 이렇게 해야 스크립트 정상적으로 돌아갈거임